src/hooks/useActiveAccount.tsx

```diff
@@ -120,43 +109,28 @@ export const ActiveAccountContextProvider = ({
     }
   }, [activeAccount?.account?.id, setStoredAccountId]);
 
-  // Clear selected account when
-  // we've detected that the userId has changed
-  useEffect(() => {
-    if (userId !== previousUserId) {
-      accountsResult.refetch();
-      setPreviousUserId(userId);
-    }
-  }, [previousUserId, userId, accountsResult]);
-
   const setActiveAccountId = useCallback(async (accountId: string) => {
     setSelectedId(accountId);
   }, []);
 
-  const refetch = useCallback(async () => {
-    return accountsResult.refetch();
-  }, [accountsResult]);
-
   // Handle invite accept
   useEffect(() => {
     const listener = async (acceptedInvite: ProjectInvite) => {
-      cache.invalidateQueries({ 'GET /v1/accounts': 'all' });
-      await refetch();
-      await setActiveAccountId(acceptedInvite.account);
+      cache.resetQueries({ 'GET /v1/accounts': 'all' });
+      setSelectedId(acceptedInvite.account);
```

Comment:
`resetQueries` is a "hard reset" -- it wipes the current value from the cache and puts us back into the "initial loading" state. By doing this, we avoid needing to refetch, and more clearly signal "we're starting over"